### ISSUE TITLE
Map is not responsive

### ISSUE THREAD

**Issue by sruthisripathi** [nicksenger/react-arcgis#44]
`import React, { Component } from 'react'; `
`import { Map } from 'react-arcgis';`

`class App extends Component {`
`  render() {`
`    return (`
`      <div style={{ width: '100vw', height: '100vh' }}>`
`       <Map /> `
`      </div>`
`   `)`
`  }`
`}`
`default export App;`

The Map does not render if I remove the div around `<Map />` component. It does not even render if I make the height and width relative: 
      `<div style={{ width: '100%', height: '100%' }}>`
       ` <Map /> ` 
      `</div>`

So, basically I am unable to make the map responsive. I want it to take the height and width of its parent div.

**Comment by nicksenger** [comment]
Taking up the full width and height of the parent container is exactly what Map does.

Percentage dimensions are relative, and do nothing unless there is some parent element with a defined size. Use width: 100vw and height: 100vh if you want to take up the full screen, or ensure html and body also have width & height of 100%.

This is not a problem with react-arcgis.
